modules/manual-configuration-image-registry-qcr.adoc

```diff
@@ -5,21 +5,31 @@
 [id="manual-configuration-image-registry-qcr_{context}"]
 = Manually configuring Quay Container Registry
 
-You can integrate {product-title} with Quay Container Registry.
+You can integrate {product-title} ({product-title-short}) with Quay Container Registry. You can integrate with Quay by using the following methods:
+
+- Integrating with the Quay public repository (registry): This method does not require authentication.
+- Integrating with a Quay private registry by using a robot account: This method requires that you create a robot account to use with Quay (recommended). See the link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/use_red_hat_quay/use-quay-manage-repo#allow-robot-access-user-repo[Quay documentation] for more information.
+- Integrating with Quay to use the Quay scanner rather than the {product-title-short} scanner:  This method uses the API and requires an OAuth token for authentication. See "Integrating with Quay Container Registry to scan images" in the "Additional Resources" section.
```

Comment:
it might help to direct link to the scanner docs here

Reply:
With our mod docs structure, we can't link to a module from another module except through this clunky "Additional Resources" section that contains the actual link. But I'll ask to see if there's another possible way.

Reply:
I checked and unfortunately my understanding is correct. I will slack you with more info as to why it's not possible.